### PR TITLE
Javascript

### PR DESCRIPTION
**Pull request by ahohnsen**

### CODE REVIEW

File: bookmarks.html [hunk 1]
```diff
@@ -6,32 +6,48 @@
...
-            data-open="Hide answer"
-            data-close="Show answer"
-          ></summary>
+        <button data-js="answer-button" class="card__answer-button">
+          <span data-js="answer-button-show">Show answer</span>
+          <span data-js="answer-button-hide" class="card__answer--hide"
+            >Hide answer</span
+          >
+        </button>
+        <p data-js="answer-text" class="card__answer--hide">
```
Comment (mwarnck) [inline bookmarks.html:44]: this is nice

File: main.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+console.clear();
```
Comment (mwarnck) [inline main.js:1]: i think you don´t need this, this was only for the codepen example to clear the console for a better view :)

**Review by mwarnck — APPROVED** [review]
good work  :)

File: main.js (hunk 2 above)
Comment (ahohnsen) [reply main.js:1]: thank you :)

**Pull request merged**